README.md

```diff
@@ -23,13 +23,20 @@ or visit our
 [Support and Help](https://www.ably.io/support)
 site to discuss privately.
 
+## Resources
+
+- [Quickstart Guide](https://www.ably.io/documentation/quick-start-guide?lang=flutter)
+- [Introducing the Ably Flutter plugin](https://www.ably.io/blog/ably-flutter-plugin) by [Srushtika](https://github.com/Srushtika) (Developer Advocate)
+- [Building a Realtime Cryptocurrency App with Flutter](https://www.ably.io/tutorials/realtime-cryptocurrency-app-flutter) by [pr-Mais](https://github.com/pr-Mais) and [escamoteur](https://github.com/escamoteur)
+- [Building realtime apps with Flutter and WebSockets: client-side considerations]()
```

Comment:
Link is missing, should be:

[Building realtime apps with Flutter and WebSockets: client-side considerations](https://www.ably.io/topic/websockets-flutter)

Reply:
Good spot. Sorry, I didn't test every link - I must have been distracted when I wrote that bullet. Will fix. 😁

Reply:
Fixed: https://github.com/ably/ably-flutter/pull/79/commits/97dcce9cd4fd75036f0e35fdf91a4128089b29a0